sqlglot/parser.py

```diff
@@ -3941,10 +3938,12 @@ def _parse_window_spec(self) -> t.Dict[str, t.Optional[str | exp.Expression]]:
 
         return {
             "value": (
-                self._match_set((TokenType.UNBOUNDED, TokenType.CURRENT_ROW)) and self._prev.text
+                (self._match_text_seq("UNBOUNDED") and "UNBOUNDED")
+                or (self._match_text_seq("CURRENT", "ROW") and "CURRENT ROW")
+                or self._parse_bitwise()
```

Comment:
We have another `self._parse_bitwise()` right below this, is this needed?

Reply:
thanks